app/services/idv/actions/verify_document_action.rb

```diff
@@ -30,9 +30,13 @@ def form
       end
 
       def enqueue_job
-        verify_document_capture_session = create_document_capture_session(
-          verify_document_capture_session_uuid_key,
-        )
+        verify_document_capture_session = if hybrid_flow_mobile?
+          document_capture_session
+        else
+          create_document_capture_session(
+            verify_document_capture_session_uuid_key,
+          )
+        end
```

Comment:
reason this is needed: in they hybrid mobile flow, the `document_capture_session` is the UUID that the desktop sent over, so that's where the desktop is polling for results. In the non-hybrid flow, we can generate a new key like this does

Reply:
the key is only used once either way, so we don't need to worry about overwriting?

Reply:
I believe, yes